src/Command/Asset/HeadAssetCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Command\Asset;
+
+use EMS\CoreBundle\Entity\UploadedAsset;
+use EMS\CoreBundle\Service\FileService;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class HeadAssetCommand extends Command
+{
+    protected FileService $fileService;
+    protected LoggerInterface $logger;
+    private SymfonyStyle $io;
+
+    public function __construct(LoggerInterface $logger, FileService $fileService)
+    {
+        $this->fileService = $fileService;
+        $this->logger = $logger;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('ems:asset:head')
```

Comment:
Fyi name is configured now via a private property: https://symfony.com/doc/current/console.html

protected static $defaultName = 'ems:asset:head';